src/test/ui/rfc-2632-const-trait-impl/cross-crate.stocknc.stderr

```diff
@@ -1,20 +1,19 @@
-error[E0277]: the trait bound `cross_crate::NonConst: cross_crate::MyTrait` is not satisfied
+error[E0277]: the trait bound `cross_crate::NonConst: cross_crate::MyTrait<_>` is not satisfied
```

Comment:
This is a diagnostic regression. `stocknc` does not use the `keyword_generics` feature and this invisible generic parameter should be hidden.